### PR TITLE
Apache 1.1 markup

### PR DESCRIPTION
**Pull request by goneall**
Resolves issue #840

### CODE REVIEW

**Comment by swinslow** [comment]
Hi @goneall, the changes to Apache-1.1.xml look great to me. 

But it looks like this PR also includes a commit to modify the matching guidelines, which doesn't look to me like it was meant to be part of addressing this issue (unless I'm misunderstanding?)  Is it possible to remove that commit, and just merge the Apache-1.1.xml changes?

**Comment by goneall** [comment]
@swinslow I'm not sure how that got into the PR - I'll update it later today after the tech call.

**Comment by goneall** [comment]
@swinslow Removed the changes to the matching guidelines.  Going to go ahead and merge once the tests pass.

File: src/Apache-1.1.xml [hunk 1]
```diff
@@ -43,18 +43,14 @@
           <alt match="name\(s\)|name|name" name="nameClause4">name</alt>
             <alt match=".+" name="organizationClause4">"Apache" and "Apache Software Foundation"</alt> must
           not be used to endorse or promote products derived from this software without prior written
-          permission. For written permission, please contact
-          <alt match=".+" name="contactClause4">apache@apache.org</alt>.
```
Comment (agr) [inline src/Apache-1.1.xml:47]: Why was the markup around the email removed?

Comment (goneall) [reply src/Apache-1.1.xml:47]: @agr The PR was done a while go, so I don't recall the reason.  Likely changing the entire sentence to optional seemed to obviate the need for the alternative clause around the contact.

There is no harm, however, in including the alternative tag as long as the optional tag is retained.

If removing this is causing an issue, feel free to create a PR adding it back.  If it causes an issue in the tests, let me know.

**Pull request merged**